### ISSUE TITLE
Downloaded translations don't match the current project configuration

### ISSUE THREAD

**Issue by KaelWD** [crowdin/github-action#60]
**Describe the bug**
Some languages are missing from our build seemingly at random. 

**To Reproduce**
Repository: https://github.com/vuetifyjs/vuetify/actions/workflows/ci.yml

This build worked: https://github.com/vuetifyjs/vuetify/runs/1959052568?check_suite_focus=true  
The next one failed on `eo` and `ja`: https://github.com/vuetifyjs/vuetify/runs/1959098539?check_suite_focus=true

**Expected behavior**
publish-docs workflow should download `eo-UY`, `ja`, and `zh-Hans` from crowdin

**Screenshots**
![image](https://user-images.githubusercontent.com/16421948/108815184-25261200-7608-11eb-9f28-3b34c95b1c1f.png)

**Additional context**
`zh-Hans` seems to always work, `ja` and `eo` randomly fail though. Seems to be most commonly just `ja` failing.

**Comment by nataliacrowdin** [comment]
Hi! Already checking everything

**Comment by nataliacrowdin** [comment]
I'm afraid we didn't reproduce this issue. We are still investigating this situation, it seems that the files were not in the branch at the moment of downloading the language eo and ja. In case this happens again - please let us know

**Comment by KaelWD** [comment]
Yeah we only store en in git, the other languages are downloaded by github actions before each build.

**Comment by MarynaAndriichuk** [comment]
Hi!
Sorry for the delay! It seems we've missed the message in the flow.
Could you please share if the behavior is still observed?

**Comment by KaelWD** [comment]
Yes, this one did it today: https://github.com/vuetifyjs/vuetify/runs/2020840737

**Comment by og-fox** [comment]
Kael, thanks for checking it! Already looking deeper into the case!

**Comment by VBeytok** [comment]
Hi @KaelWD, please add to each downloaded language parameter `debug_mode: true` in your `ci.yml` workflow.

**Comment by KaelWD** [comment]
Sorry I missed this

@VBeytok `Unexpected input(s) 'debug_mode', valid inputs are ['entryPoint', 'args', ...`

**Comment by KaelWD** [comment]
@VBeytok @og-fox has there been any progress on this?

**Comment by Andrulko** [comment]
@KaelWD It may be caused by default API limitation (projects can be built once per 30 minutes, if you trigger builds more frequently - there is a chance that the next build will be skipped). It is now disabled for your project, please let us know if the problem is gone.

Commonly, this limitation is not even noticeable but it may occur if project builds are quite frequent.

You can test it this way:

1) Add a translation;
2) Run `crowdin download` or `crowdin download -b branch_name`
3) Add another translation after download is finished;
4) Run `crowdin download` again. The project build should not be skipped this time.

Thanks,

**Comment by KaelWD** [comment]
That makes sense, looking back it does seem to happen when there's a few CI runs close together. Having a rate limit is always a good idea, but I think some better messaging would be good because currently if you hit the limit it just says "Downloaded translations don't match the current project configuration" with no reason given. If the limit is only for building the project, having it cache builds for 30min would also be acceptable.

**Comment by KaelWD** [comment]
Was this intentional? I now get a runtime exception when there's too many CI runs close together:
![image](https://user-images.githubusercontent.com/16421948/120356996-4c4b5880-c348-11eb-8698-50a67ddeabe9.png)

**Comment by andrii-bodnar** [comment]
Hi @KaelWD,

does this message contains the `Code: 409, Message: Another build is currently in progress. Please wait until it's finished.` response?

If so, then this is expected behavior because parallel downloads are not allowed by Crowdin.

**Comment by KaelWD** [comment]
Ah yep. I think we'll need to switch to a daily cached crowdin build then.

**Comment by Natalia936** [comment]
Hi @KaelWD,

Sure, you're welcome to try it out 😉

**Comment by KaelWD** [comment]
Would be nice if it was built-in though, no need to build the project again if nothing has changed since last time.

**Comment by Natalia936** [comment]
Hi @KaelWD,

Sorry for the delayed answer, however, we're wondering if the inconvenience with the build was fixed on your side?

**Comment by lixw1021** [comment]
Any update on this, I had the same issue with my project?

**Comment by Olena1234** [comment]
Hi there! Previously, you have mentioned that you will switch to a daily cached crowdin build, could you please let us know if you did that?

**Comment by andrii-bodnar** [comment]
https://github.com/crowdin/github-action/wiki/Handling-parallel-runs
